packages/eas-cli/src/build/ios/prepareJob.ts

```diff
@@ -45,9 +46,16 @@ export async function prepareJobAsync(
     }
   }
 
-  const maybeCustomBuildConfigPath = buildProfile.config
+  let maybeCustomBuildConfigPath = buildProfile.config
     ? getCustomBuildConfigPath(buildProfile.config)
     : undefined;
+  if (
+    maybeCustomBuildConfigPath &&
+    ctx.localBuildOptions.localBuildMode !== LocalBuildMode.LOCAL_BUILD_PLUGIN &&
+    process.platform === 'win32'
+  ) {
+    maybeCustomBuildConfigPath = path.posix.join(...maybeCustomBuildConfigPath.split(path.sep));
```

Comment:
```suggestion
    maybeCustomBuildConfigPath = path.posix.normalize(maybeCustomBuildConfigPath);
```
would this have the same effect?

Reply:
Pity. I noticed it works the other way

```
> path.win32.normalize("d/b/a.yaml")
'd\\b\\a.yaml'
```
and I hoped it would go the other way too, but it doesn't.

```
> path.posix.normalize("d\\b\\a.yaml")
'd\\b\\a.yaml'
```